webui/react/src/hooks/useTheme.ts

```diff
@@ -7,6 +7,7 @@ import { BrandingType } from 'types';
 
 type ThemeHook = {
   mode: DarkLight,
+  scheme: DarkLight,
   setBranding: Dispatch<SetStateAction<BrandingType>>,
   setMode: Dispatch<SetStateAction<DarkLight>>,
   theme: Theme,
```

Comment:
**chore:** So I think the code around detecting the system needs to get updated to support detection of `Auto` mode. Currently we can only distinguish between `Dark` vs `Light / Auto`. We can't tell `Light` from `Auto`. To do that, we'll need to do something like this:
```
const MATCH_MEDIA_SCHEME_DARK = '(prefers-color-scheme: dark)';
const MATCH_MEDIA_SCHEME_LIGHT = '(prefers-color-scheme: light)';
```
and instead of `getIsDarkMode`, use something like this:
```
export enum Mode {
  Auto = 'auto',
  Dark = DarkLight.Dark,
  Light = DarkLight.Light,
}

const getSystemMode = (): Mode => {
  const isDark = matchMedia?.(MATCH_MEDIA_SCHEME_DARK).matches;
  if (isDark) return Mode.Dark;

  const isLight = matchMedia?.(MATCH_MEDIA_SCHEME_LIGHT).matches;
  if (sLight) return Mode.Light;

  return Mode.Auto;
};
```